assets/src/dashboard/components/typeaheadOptions/index.js

```diff
@@ -28,36 +28,40 @@ import { KEYS, Z_INDEX } from '../../constants';
 import { DROPDOWN_ITEM_PROP_TYPE } from '../types';
 
 export const Menu = styled.ul`
+  ${({ theme, isOpen }) => `
   width: 100%;
   align-items: flex-start;
-  background-color: ${({ theme }) => theme.colors.white};
+  background-color: ${theme.colors.white};
+  box-shadow: ${theme.expandedTypeahead.boxShadow};
+  border-radius: 5px;
   display: flex;
   flex-direction: column;
-  margin: 0;
-  opacity: ${({ isOpen }) => (isOpen ? 1 : 0)};
+  margin: 5px 0 0;
+  opacity: ${isOpen ? 1 : 0};
   overflow: hidden;
-  padding: 0;
-  pointer-events: ${({ isOpen }) => (isOpen ? 'auto' : 'none')};
+  padding: 5px 0;
+  pointer-events: ${isOpen ? 'auto' : 'none'};
   z-index: ${Z_INDEX.TYPEAHEAD_OPTIONS};
+`}
```

Comment:
Same super duper nit as below, the indentation of the styles is missing 😱

Reply:
Fixed - but do you know why automation missed this? Is there something I need to do to get styled components + indenting to work when passing props this way as opposed to by style?

Reply:
Hmmmm... I'm guess it got confused because everything is wrapped in the string interpolation ticks and maybe it's not setup to format those 🤔